securityconfig/roles.yml

```diff
@@ -2,6 +2,13 @@ _meta:
   type: "roles"
   config_version: 2
 
+# Restrict users so they can only view visualization and dashboard on kibana
+kibana_read_only:
+  reserved: true
+
+rest_api_access:
```

Comment:
we use `security_rest_api_access` in the elasticsearch.yml. Lets add the following comment. "The security REST API access role is used to assign specific users access to change the security settings through the REST API."

Reply:
Updated the PR to reflect this change.